.github/workflows/event-merge-to-queue.yml

```diff
@@ -57,47 +23,41 @@ jobs:
   docs-only: # Check whether the PR is only modifying docs
     uses: ./.github/workflows/task-check-docs.yml
 
-  test-linux:
-    needs: [print-configurations, docs-only, get-latest-redis-tag]
+  # Add a matrix with the different Redis versions to test against here when needed
+  test-linux-x86_64:
+    needs: [docs-only, get-latest-redis-tag]
     if: needs.docs-only.outputs.only-docs-changed == 'false'
-    strategy:
-      matrix:
-        test-config: # configurations for both standalone and coordinator
-          ${{ fromJson(needs.print-configurations.outputs.test-configurations) }}
-        include: # spacial configurations
-          ${{ fromJson(needs.print-configurations.outputs.include-configurations) }}
+    uses: ./.github/workflows/flow-linux-platforms.yml
```

Comment:
So by using this workflow, we get all the flows we would have gotten from the matrix we no longer use, right? (Same for linux aarch64)

Reply:
This matrix actually had only one job. It was a preparation for parallelism but it's not beneficial